src/test/java/org/folio/marc4ld/authority/field110/MarcToBibframeAuthorityEmptySubfocus110IT.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.marc4ld.authority.field110;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+
+import org.folio.marc4ld.Marc2LdTestBase;
+import org.junit.jupiter.api.Test;
+
+class MarcToBibframeAuthorityEmptySubfocus110IT extends Marc2LdTestBase {
```

Comment:
We should use "ld" term instead of "bibframe" at least for new classes. Old ones will be also renamed at some point of time.

Reply:
Fixed.